tools/link/README.md

```diff
@@ -0,0 +1,23 @@
+# `@lg-tools/link`
+
+Link and Unlink packages to a consuming repository
+
+## Usage
+
+Use from `@lg-tools/cli`.
+
+### Link
+
+Run `link` from the leafygreen repository to link packages to a consuming app.
```

Comment:
does this work from all of our repos?

Reply:
Yep! It'll link all workspaces defined in `lg.config`